sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UseStringIsNullOrEmpty.cs

```diff
@@ -51,34 +50,25 @@ protected override void Initialize(SonarAnalysisContext context)
                 {
                     var invocationExpression = (InvocationExpressionSyntax)c.Node;
 
-                    if (!(invocationExpression.Expression is MemberAccessExpressionSyntax memberAccessExpression))
+                    if (invocationExpression.Expression is MemberAccessExpressionSyntax memberAccessExpression &&
+                        TryGetFirstArgument(invocationExpression, out var firstArgument) &&
```

Comment:
To follow our coding style:
```suggestion
                        && TryGetFirstArgument(invocationExpression, out var firstArgument) 
```

Reply:
While refactoring, you can fix this in whole file